rust-cross-template.spec

```diff
@@ -15,7 +15,7 @@
 %global rel %(rpm -qi rust-std-static-%{rust_x86_triple} | grep Release | cut -f2 -d:)
 
 Name:          rust-cross
-Version:       1.0+git2
+Version:       1.0+git3
 Release:       1
 Source10:      precheckin.sh
```

Comment:
Ah, host-gcc-wrapper needs to be also on Source:

Reply:
Fixed